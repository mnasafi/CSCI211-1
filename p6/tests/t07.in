point 33 2 .
point 34 2 '
point 36 2 `
point 37 2 '
point 38 2 .
point 39 2 _
point 40 2 _
point 32 3 /
point 39 3 \
point 41 3 `
point 42 3 '
point 43 3 "
point 44 3 -
point 45 3 ,
point 15 4 .
point 16 4 -
point 17 4 '
point 18 4 '
point 19 4 '
point 20 4 '
point 21 4 -
point 22 4 -
point 23 4 .
point 24 4 .
point 25 4 .
point 26 4 _
point 27 4 _
point 28 4 .
point 29 4 .
point 30 4 -
point 31 4 /
point 33 4 .
point 39 4 |
point 46 4 \
point 13 5 .
point 14 5 '
point 30 5 ;
point 32 5 :
point 33 5 '
point 39 5 '
point 40 5 .
point 43 5 a
point 47 5 |
point 12 6 /
point 30 6 |
point 32 6 :
point 33 6 .
point 41 6 \
point 47 6 =
point 48 6 \
point 11 7 ;
point 31 7 \
point 32 7 '
point 33 7 :
point 34 7 .
point 41 7 /
point 44 7 ,
point 45 7 -
point 46 7 .
point 47 7 _
point 48 7 _
point 49 7 ;
point 50 7 .
point 51 7 -
point 52 7 ;
point 53 7 `
point 10 8 /
point 11 8 |
point 17 8 .
point 32 8 '
point 33 8 -
point 34 8 -
point 35 8 .
point 36 8 _
point 40 8 /
point 41 8 -
point 42 8 .
point 43 8 7
point 44 8 `
point 45 8 .
point 46 8 _
point 47 8 .
point 48 8 .
point 49 8 -
point 50 8 ;
point 51 8 `
point 9 9 ;
point 11 9 |
point 19 9 '
point 36 9 |
point 37 9 `
point 38 9 -
point 39 9 '
point 46 9 \
point 49 9 =
point 50 9 |
point 9 10 |
point 10 10 /
point 11 10 \
point 20 10 .
point 24 10 -
point 25 10 '
point 27 10 /
point 33 10 /
point 36 10 ;
point 46 10 |
point 49 10 =
point 50 10 /
point 9 11 (
point 10 11 (
point 12 11 ;
point 13 11 .
point 21 11 ,
point 22 11 _
point 25 11 .
point 26 11 :
point 27 11 |
point 33 11 |
point 35 11 /
point 41 11 /
point 42 11 \
point 46 11 |
point 48 11 =
point 49 11 |
point 10 12 )
point 12 12 /
point 14 12 `
point 15 12 \
point 21 12 |
point 23 12 `
point 24 12 "
point 25 12 "
point 26 12 `
point 27 12 ;
point 33 12 /
point 35 12 |
point 40 12 |
point 42 12 /
point 46 12 /
point 48 12 =
point 49 12 /
point 12 13 |
point 14 13 :
point 15 13 :
point 16 13 |
point 21 13 |
point 28 13 \
point 33 13 \
point 35 13 \
point 40 13 \
point 42 13 `
point 43 13 -
point 44 13 -
point 45 13 '
point 47 13 =
point 48 13 /
point 11 14 /
point 14 14 '
point 15 14 /
point 16 14 \
point 21 14 /
point 29 14 )
point 34 14 |
point 35 14 /
point 41 14 `
point 42 14 -
point 43 14 .
point 44 14 .
point 45 14 .
point 46 14 -
point 47 14 `
point 10 15 /
point 15 15 |
point 17 15 |
point 20 15 `
point 21 15 \
point 26 15 /
point 27 15 -
point 28 15 '
point 33 15 /
point 34 15 ;
point 10 16 \
point 13 16 ,
point 14 16 ,
point 15 16 /
point 17 16 |
point 22 16 \
point 26 16 D
point 31 16 .
point 32 16 '
point 35 16 \
point 11 17 `
point 12 17 "
point 13 17 "
point 14 17 `
point 18 17 \
point 21 17 n
point 22 17 n
point 23 17 h
point 26 17 D
point 27 17 _
point 28 17 .
point 29 17 -
point 30 17 '
point 31 17 L
point 32 17 _
point 33 17 _
point 34 17 n
point 35 17 n
point 36 17 h
point 19 18 `
point 20 18 "
point 21 18 "
point 22 18 "
point 23 18 `
point 0 21 s
point 1 21 o
point 2 21 u
point 3 21 r
point 4 21 c
point 5 21 e
point 6 21 :
point 8 21 w
point 9 21 w
point 10 21 w
point 11 21 .
point 12 21 a
point 13 21 l
point 14 21 b
point 15 21 i
point 16 21 n
point 17 21 o
point 18 21 b
point 19 21 l
point 20 21 a
point 21 21 c
point 22 21 k
point 23 21 s
point 24 21 h
point 25 21 e
point 26 21 e
point 27 21 p
point 28 21 .
point 29 21 c
point 30 21 o
point 31 21 m
point 32 21 /
point 33 21 t
point 34 21 x
point 35 21 t
point 36 21 /
point 37 21 a
point 38 21 s
point 39 21 c
point 40 21 i
point 41 21 i
point 42 21 .
point 43 21 p
point 44 21 h
point 45 21 p
