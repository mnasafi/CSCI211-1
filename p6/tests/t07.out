                                                            
                                                            
                                 .' `'.__                   
                                /      \ `'"-,              
               .-''''--...__..-/ .     |      \             
             .'               ; :'     '.  a   |            
            /                 | :.       \     =\           
           ;                   \':.      /  ,-.__;.-;`      
          /|     .              '--._   /-.7`._..-;`        
         ; |       '                |`-'      \  =|         
         |/\        .   -' /     /  ;         |  =/         
         (( ;.       ,_  .:|     | /     /\   | =|          
          ) / `\     | `""`;     / |    | /   / =/          
            | ::|    |      \    \ \    \ `--' =/           
           /  '/\    /       )    |/     `-...-`            
          /    | |  `\    /-'    /;                         
          \  ,,/ |    \   D    .'  \                        
           `""`   \  nnh  D_.-'L__nnh                       
                   `"""`                                    
                                                            
                                                            
source: www.albinoblacksheep.com/txt/ascii.php              
                                                            
                                                            
